panel/src/main/resources/inst_data_2015_20170309_dummy.csv

```diff
@@ -1,213 +1,213 @@
-2015|-1|-1|-1|-1|0|1609343|0|0|0||||GREYLOCK PIPELINE INC.|TX|HOUSTON|48|0|NaN|||||-1|-1
-2015|-1|-1|-1|-1|0|3318881|0|0|0||||SOUTHFORK CLO LTD.|TX|HOUSTON|48|0|NaN|||||-1|-1
-2015|-1|-1|-1|-1|0|1041103|0|0|0||||INTERACTIVE DATA CORPORATION|MA|WALTHAM|25|0|NaN|||||-1|-1
-2015|-1|-1|-1|-1|0|1582156|0|0|0||||BOLIVAR INTERNATIONAL BANK|0|GEORGE TOWN|0|0|NaN|||||-1|-1
-2015|-1|-1|-1|-1|0|1556942|0|0|0||||EQUITABLE OF IOWA COMPANIES|IA|DES MOINES|19|0|NaN|||||-1|-1
-2015|-1|-1|11|-1|371456845|4292689|0|0|0||||GOLD LEAF TITLE COMPANY|FL|MIAMI|12|0|21468|790534.0|TOTALBANK|MIAMI|FL|-1|2
-2015|-1|-1|11|-1|371456845|4292689|0|0|0||||GOLD LEAF TITLE COMPANY|FL|MIAMI|12|0|21468|790534.0|TOTALBANK|MIAMI|FL|-1|2
-2015|-1|-1|-1|-1|205652788|3484405|0|0|0||||CREDIT SUISSE/CFIG CERBERUS IV SPV, LLC|NY|NEW YORK|36|0|NaN|||||-1|-1
-2015|18724|3|3|-1|470125910|235558|18724|0|0||||CITIZENS STATE BANK|NE|CARLETON|31|0|-1|1052949.0|CARLETON AGENCY, INC.|CARLETON|NE|17267|0
-2015|-1|-1|-1|-1|0|2548588|0|0|0||||SOUND ADVICE, INC.|FL|DANIA|12|0|NaN|||||-1|-1
-2015|-1|-1|-1|-1|0|1653177|0|0|0||||BRAMALEA CAPITAL CORP.|0|TORONTO|0|0|NaN|||||-1|-1
-2015|-1|-1|-1|-1|200740151|3872365|0|0|0||||UNITED COMMUNITY MORTGAGE CORP|NJ|KEYPORT|34|0|NaN|||||-1|-1
-2015|-1|-1|-1|-1|0|2458436|0|0|0||||ITAUTEC PHILCO S.A.|0|MANAUS|0|0|NaN|||||-1|-1
-2015|-1|-1|-1|-1|0|4563307|0|0|0||||FACULTAS FUND (GENPAR), LLC|NY|NEW YORK|36|0|-1|2295109.0|BLACKROCK FINANCIAL MANAGEMENT, INC.|NEW YORK|NY|-1|-1
-2015|-1|-1|9|-1|330917586|3825978|0|0|0||||LONG BEACH SECURITIES CORP.|CA|ORANGE|6|0|852218|852218.0|JPMORGAN CHASE BANK, NATIONAL ASSOCIATION|COLUMBUS|OH|-1|1
-2015|-1|-1|9|-1|330917586|3825978|0|0|0||||LONG BEACH SECURITIES CORP.|CA|ORANGE|6|0|852218|852218.0|JPMORGAN CHASE BANK, NATIONAL ASSOCIATION|COLUMBUS|OH|-1|1
-2015|-1|-1|-1|-1|0|4867322|0|0|0||||PARADEPLATZ 2002 PLAN, L.P.|0|HAMILTON|0|0|-1|2803924.0|CREDIT SUISSE (BERMUDA) LIMITED|NEW YORK|NY|-1|-1
-2015|-1|-1|-1|-1|0|4867322|0|0|0||||PARADEPLATZ 2002 PLAN, L.P.|0|HAMILTON|0|0|-1|2803924.0|CREDIT SUISSE (BERMUDA) LIMITED|NEW YORK|NY|-1|-1
-2015|-1|-1|-1|-1|0|4403636|0|0|0||||BLACKROCK FIN STRATEGIC INCOME OPPORTUNITIES BOND FUND|CA|SAN FRANCISCO|6|0|NaN|||||-1|-1
-2015|-1|-1|-1|-1|0|2710910|0|0|0||||ENVIRONMENTAL PRIVATE EQUITY MANAGEMENT II, L.P.|CA|SAN FRANCISCO|6|0|NaN|||||-1|-1
+year|rid|agency|inst_type|cra_filer|id_tax|id_rssd|id_fdic_cert|id_ncua|id_occ|domain_2015|domain_2014|domain_2013|nm_lgl|state_abbr_nm|city|state_cd|hmda_filer|p_rid|p_rssd|p_name|p_city|p_state|assets|olc|top_holder|top_holder_name|top_holder_city|top_holder_state|top_holder_country
+2015|Bank0_RID|9|1|1|b0-testtaxid|6999998|7999998|8999998|9999998|@bank0.com|@bank0.com|@bank0.com|bank-0 National Association|CA|Davis|06|1|Bank0_Parent_RID|6999988.0|Bank0-parent|Somewhereville|CA|100000|0|-1||||
```

Comment:
@nickgrippin - can you make this change for this PR, for bank 0 and bank 1? @Kibrael has an issue open to fix it on his end too.